src/pages/lib/graphqlapi/query-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,604 @@ export const meta = {
   title: `Read application data`,
   description: `Learn more about how to read data using Amplify's GraphQL API category`
 };
+
+<InlineFilter filters={["js", "react-native"]}>
+
+You can read application data using the Amplify GraphQL client. In this guide we will review the difference between reading data and getting data, how to filter query results to get just the data you need, and how to paginate results to make your data more manageable. We will also show you how to cancel these requests when needed.
+
+Before you begin, you will need:
+
+<InlineFilter filters={["js"]}>
+
```

Comment:
Noting here again that js is linking to the new page so will want to consider as we push this to main.